new-collector-page-test/winter-collector-page-test.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
Need to write unit tests for the function that you will create to generate the HTML of all the cats.